dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeAPIImpl.java

```diff
@@ -116,7 +116,7 @@ public void delete(ContentType type) throws DotSecurityException, DotDataExcepti
   @Override
   @CloseDBIfOpened
   public ContentType find(final String inodeOrVar) throws DotSecurityException, DotDataException {
-
+    if(!UtilMethods.isSet(inodeOrVar))return null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Missing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00122)